File: Tema 2/Ejercicios/Ejercicio Tema 2 Bloque 0/Ejercicio 1.py
numero1=int(input("Dime un numero: "))
numero2=int(input("Dime un numero: "))
numero3=int(input("Dime un numero: "))
if numero1 < numero2 and numero2 < numero3:
    print("El menor es el numero", numero1)
if numero2 < numero1 and numero2 < numero3:
    print("El menor es el numero", numero2)
if numero3 < numero2 and numero3 < numero1:
    print("El menor es el numero", numero3)   

File: Java/Ejercicios Repaso java 1/Ejercicios repaso 1/src/Ejercicio_1.java
import java.util.Scanner;

public class Ejercicio_1 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Dime un numero: ");
        int numero1 = scanner.nextInt();
        System.out.print("Dime un numero: ");
        int numero2 = scanner.nextInt();
        System.out.print("Dime un numero: ");
        int numero3 = scanner.nextInt();
        System.out.print("Dime un numero: ");
        int numero4 = scanner.nextInt();
        int mayor = Math.max(numero1, numero2);
        int mayo2 = Math.max(numero3, numero4);
        int mayordefinitivo = Math.max(mayor, mayo2);
        if (numero1 == numero2 || numero1 == numero3 || numero1 == numero4) {
            System.out.println("Los numeros son iguales");
            System.exit(1);
        }
        if (numero2 == numero3 || numero2 == numero4) {
            System.out.println("Los numeros son iguales");
            System.exit(1);
        }
        if (numero3 == numero4) {
            System.out.println("Los numeros son iguales");
            System.exit(1);
        }
        if (numero4 == numero3) {
            System.out.println("Los numeros son iguales");
            System.exit(1);
        } else {
            System.out.println(mayordefinitivo);
        }
    }
}
